src/Core/Framework/Adapter/Filesystem/Plugin/CopyBatchInput.php

```diff
@@ -2,9 +2,13 @@
 
 namespace Shopware\Core\Framework\Adapter\Filesystem\Plugin;
 
+use League\Flysystem\Visibility;
 use Shopware\Core\Framework\Adapter\AdapterException;
 use Shopware\Core\Framework\Log\Package;
 
+/**
+ * @final
```

Comment:
I am so sorry :see_no_evil: but wouldn't that be a break, if we just say, it is final? :see_no_evil:

Reply:
any better idea? 😂

Reply:
not needed symfony is doing it https://github.com/symfony/symfony/blob/7.4/src/Symfony/Component/ErrorHandler/DebugClassLoader.php#L661